### DISCUSSION TITLE
posts/2023/10/direct-deployments-quicker-slicker/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#36] [Announcements]
# posts/2023/10/direct-deployments-quicker-slicker/

Providers’ free tiers are limited, so why not rely on the computing power you already own?

https://www.brycewray.com/posts/2023/10/direct-deployments-quicker-slicker/?utm_campaign=RSS&utm_source=RSS&utm_medium=RSS

**Comment by bobmonsour** [comment]
Well that's a cool way to deploy! I always waste time watching Netlify do its build to make sure it doesn't fail (not that it does much). But this looks pretty sweet and makes a lot of sense for some situations. Thanks for describing it all.
